node/Cargo.toml

```diff
@@ -202,7 +202,7 @@ git = "https://github.com/paritytech/polkadot"
 branch = 'rococo-v1'
 
 [[bin]]
-name = 'parachain-collator'
+name = 'pint'
```

Comment:
We can just delete this 

```toml
[[bin]]
name = "pint"
```

As we already have

```toml
[package]
name = "pint"
```

Reply:
Good catch